purchase_order_line_description_search/views/purchase_order_views.xml

```diff
@@ -0,0 +1,30 @@
+<odoo>
+    <record id="view_purchase_order_filter" model="ir.ui.view">
+        <field name="name">request.quotation.select</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.view_purchase_order_filter" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='product_id']" position="after">
```

Comment:
@kanda999 Please check with the client about the position. I thought they might want the field as the first option among others.